src/Components/Weather.jsx

```diff
@@ -8,20 +8,72 @@ import search_icon from '../Assets/search.png';
 import snow_icon from '../Assets/snow.png';
 import rain_icon from '../Assets/rain.png';
 import clear_icon from '../Assets/clear.png';
+import { useState } from 'react';
 
 
 export const Weather = () => {
+    let api_key = '64e27350a7f1134f0c9bfd6d95c4f1a8';
+
```

Comment:
Kindly make sure that your api is not exposed. instead use environment variables to keep it secure